File: ProgII/src/Model/Produto.java
package Model;

public class Produto{
	
	public static final String PRODUTO_COD   ="idProduto";
	public static final String PRODUTO_NOME  ="nome";
	public static final String PRODUTO_DESC  ="descricao";
	public static final String PRODUTO_PRECO ="valor";
	public static final String PRODUTO_SALDO ="saldo";
	public static final String PRODUTO_CATEG ="idCategoria";
	public static final String PRODUTO_MARCA ="idMarca";
	public static final String PRODUTO_IMAGEM ="imagem";
	public static final String PRODUTO_U_M = "unidadeMedida";
	
	
	private int idProduto;
	private String nomeProduto;
	private String descricaoProduto;
	private float precoVenda;
	private float estoque;
	private String unidadeMedida;
	private Categoria categoria;
	private Marca marca;
	private String imagem;
	
	
	public int getIdProduto()
	{
		return idProduto;
	}
	public void setIdProduto(int idProduto)
	{
		this.idProduto = idProduto;
	}
	public String getNomeProduto()
	{
		return nomeProduto;
	}
	public void setNomeProduto(String nomeProduto)
	{
		this.nomeProduto = nomeProduto;
	}
	public String getDescricaoProduto()
	{
		return descricaoProduto;
	}
	public void setDescricaoProduto(String descricaoProduto)
	{
		this.descricaoProduto = descricaoProduto;
	}
	public double getPrecoVenda()
	{
		return precoVenda;
	}
	public void setPrecoVenda(float precoVenda)
	{
		this.precoVenda = precoVenda;
	}
	public float getEstoque()
	{
		return estoque;
	}
	public void setEstoque(float estoque)
	{
		this.estoque = estoque;
	}
	
	public Categoria getCategoria()
	{
		return categoria;
	}
	public void setCategoria(Categoria categoria)
	{
		this.categoria = categoria;
	}
	public Marca getMarca()
	{
		return marca;
	}
	public void setMarca(Marca marca)
	{
		this.marca = marca;
	}
	public String getImagem()
	{
		return imagem;
	}
	public void setImagem(String imagem)
	{
		this.imagem = imagem;
	}
	public String getUnidadeMedida() {
		return unidadeMedida;
	}
	public void setUnidadeMedida(String unidadeMedida) {
		this.unidadeMedida = unidadeMedida;
	}
	
	
	
	
	
}

File: Sistema_Web/models/Produto.php
<?php

require_once 'Marca.php';
require_once 'Categoria.php';


class Produto{
    private $idProduto;
    private $nome;
    private $descricao;
    private $saldo;
    private $Categoria;
    private $Marca;
    private $valor;
    private $ano;
    private $imagem;



    /**
     * Produtp constructor.
     * @param $idProduto
     */
    public function __construct()
    {
        $this->idProduto = 0;
        $this->Categoria = new Categoria();
        $this->Marca = new Marca();
    }


    /**
     * @return mixed
     */
    public function getIdProduto()
    {
        return $this->idProduto;
    }

    /**
     * @param mixed $idProduto
     */
    public function setIdProduto($idProduto)
    {
        $this->idProduto = $idProduto;
    }

    /**
     * @return mixed
     */
    public function getNome()
    {
        return $this->nome;
    }

    /**
     * @param mixed $nome
     */
    public function setNome($nome)
    {
        $this->nome = $nome;
    }

    /**
     * @return mixed
     */
    public function getDescricao()
    {
        return $this->descricao;
    }

    /**
     * @param mixed $descricao
     */
    public function setDescricao($descricao)
    {
        $this->descricao = $descricao;
    }

    /**
     * @return mixed
     */
    public function getSaldo()
    {
        return $this->saldo;
    }

    /**
     * @param mixed $saldo
     */
    public function setSaldo($saldo)
    {
        $this->saldo = $saldo;
    }

    /**
     * @return mixed
     */
    public function getCategoria()
    {
        return $this->Categoria;
    }

    /**
     * @param mixed $Categoria
     */
    public function setCategoria($Categoria)
    {
        $this->Categoria = $Categoria;
    }

    /**
     * @return mixed
     */
    public function getMarca()
    {
        return $this->Marca;
    }

    /**
     * @param mixed $Marca
     */
    public function setMarca($Marca)
    {
        $this->Marca = $Marca;
    }

    /**
     * @return mixed
     */
    public function getValor()
    {
        return $this->valor;
    }

    /**
     * @param mixed $valor
     */
    public function setValor($valor)
    {
        $this->valor = $valor;
    }

    /**
     * @return mixed
     */
    public function getAno()
    {
        return $this->ano;
    }

    /**
     * @param mixed $ano
     */
    public function setAno($ano)
    {
        $this->ano = $ano;
    }

    /**
     * @return mixed
     */
    public function getImagem()
    {
        return $this->imagem;
    }

    /**
     * @param mixed $imagem
     */
    public function setimagem($imagem)
    {
        $this->imagem = $imagem;
    }

}